packages/lib/services/interop/InteropService_Exporter_Md.ts

```diff
@@ -83,17 +85,18 @@ export default class InteropService_Exporter_Md extends InteropService_Exporter_
 				notePaths: {},
 			};
 			for (let i = 0; i < itemsToExport.length; i++) {
-				const itemType = itemsToExport[i].type;
+				const it = itemsToExport[i].type;
 
-				if (itemType !== itemType) continue;
+				if (it !== itemType) continue;
```

Comment:
This was a bug causing the below code to never run (or always run I can't remember now)